airbyte-api/src/main/openapi/config.yaml

```diff
@@ -1097,10 +1097,14 @@ components:
           type: string
         initialSetupComplete:
           type: boolean
-        onboardingComplete:
-          type: boolean
         displaySetupWizard:
           type: boolean
+        anonymousDataCollection:
+          type: boolean
+        news:
```

Comment:
The name of this flag is not clear. Maybe "subscribedToNewsletter" or something? Same for the security flag

Reply:
that's a bigger change at this point. this attribute exists in many places already.